docs/installation/development.rst

```diff
@@ -100,14 +100,19 @@ so you can easily edit them with your favorite IDE.
 3. `cd ploneintranet`
 4. `sudo make docker-build` will prepare the docker image
 5. `sudo make docker-run` will launch the docker image and give you a shell inside
-6. `make` will fetch the ploneintranet compiled javascript bundle
-7. `make devel` will run the development buildout
-8. `bin/instance fg` will run the Plone instance, which you can access on localhost:8080.
+
+.. note:: Make sure you are inside the shell (usually your prompt changes!)
```

Comment:
"shell" is not the right word here. All these commands are shell commands. Should be "inside the docker container".